### PR TITLE
Matrix plugin.audio.pureradio 3.0.2

### PR DESCRIPTION
**Pull request by bdommel**
Update for Kodi 19+
- Complete rewrite based on guidelines for Kodi
- Python3 support
- Podcast section optimalisations. Retrieval is much faster and optional (via add-on settings) artwork can be shown.

Many thanks to the Kodi Team for their support and making this awesome application possible.

### CODE REVIEW

**Comment by learningit** [comment]
You need to squash all these commits down into a single commit for submission. Please have a look at this for how to do it:

https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

**Comment by bdommel** [comment]
As far as I can see, the squashing of the commits is done.

**Comment by enen92** [comment]
It is not, still 15 commits but don't worry about it. We can squash them when merging.

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,282 @@
+
```
Comment (enen92) [inline LICENSE.txt:1]: This file is not contained within your addon folder

File: README.md [hunk 2]
```diff
@@ -1,24 +0,0 @@
-![Kodi logo](https://raw.githubusercontent.com/xbmc/xbmc/master/media/banner.png)
```
Comment (enen92) [inline README.md:1]: Please fix your pull request

**Review by enen92 — CHANGES_REQUESTED** [review]
Actually you should care. You're removing files from the matrix repository and adding files outside the folder of your addon...

**Comment by bdommel** [comment]
This squashing of the commits gave strange results. Still trying to see why I got a commit history on a new branche with a single commit. I will do a new commit and pull request.

**Pull request closed without merging**